.editorconfig

```diff
@@ -30,6 +30,7 @@ indent_size = 2
 
 [*.{yml,yaml}]
 indent_style = space
+indent_size = 2
```

Comment:
ℹ️ I added this per discussion. If you'd like it to be a separate commit, I can remove and do a separate PR; totally fine either way.